### ISSUE TITLE
2.0 release

### ISSUE THREAD

**Issue by adamralph** [adamralph/xbehave.net#144]
**Ready** when all other issues forming part of the release are **Done**.
- [x] ensure patches have been merged from master to dev
- [x] run code analysis in VS in _Release_ mode and address violations (send a regular PR which must be merged before continuing)

---
- [x] RC smoke tests (passes, various types of faliures, skips)
  - [x] console - console output + html
  - [x] VS runner
  - [x] msbuild
  - [x] testdriven.net
  - [x] R#
  - [x] ncrunch
- [x] RC test update in projects
  - ~~scriptcs~~ blocked by AutoFixture.Xunit
  - [x] ConfigR
  - ~~Bau~~ Bau.Xunit does not yet support xunit 2.0

---
- [x] prepare wiki
  - [x] refresh current pages
  - [x] add pages for new features
- [x] prepare blog post
- [x] prepare website

---
- [x] create and merge PR from upstream dev to master
- [x] smoke test NuGet package
- [x] create release in GitHub UI including releaseNotes, mentioning non-owner contributors, if any
- [x] push nuget package
- [x] copy release notes from GitHub to NuGet
- [x] de-list buggy nuget packages if present
- [x] publish blog post
- [x] publish website
- [x] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
- [x] post tweet in JabbR (xbehavenet and general-chat)
- [x] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
- [x] increment minor version
- [x] create and merge PR to upstream dev
- [x] update samples/raise issue to do so
- [x] create a new milestone for the next release
- [x] create new issue based on this one for next release, adding it to the new milestone
- [x] close all issues on this milestone
- [x] close this milestone

**Comment by adamralph** [comment]
There is some feedback on 2.0 RC1 over at #51

**Comment by adamralph** [comment]
https://twitter.com/adamralph/status/629330749251055616
